falco-exporter/README.md

```diff
@@ -73,7 +76,3 @@ helm install falco-exporter -f values.yaml falcosecurity/falco-exporter
 ```
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
-
-## Using falco-exporter with Grafana
-
-See the [Grafana section](https://github.com/falcosecurity/falco-exporter#grafana) in the [falco-exporter](https://github.com/falcosecurity/falco-exporter) documentation.
```

Comment:
Could you document how to enable the dashboard and what requirements are needed to make it work?

Reply:
Isn't that literally done in the value part of the readme?

Reply:
Added the url over there

Reply:
Thank you, I will try this PR as soon as I can

Reply:
/assign